README.md

```diff
@@ -24,14 +24,9 @@ The recommended way to install the `enry` command-line tool is to either
 [download a release](https://github.com/src-d/enry/releases) or run:
 
 ```
-(cd /tmp; go get github.com/src-d/enry/v2/cmd/enry)
+(cd "$(mktemp -d)"; go mod init enry; go get github.com/src-d/enry/v2/cmd/enry)
```

Comment:
Why not concatenate the commands with `&&` instead of `;`?

Reply:
No good reason, just because it was that way before.